providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/connector/grpc/GrpcStreamConnector.java

```diff
@@ -50,6 +58,7 @@ public class GrpcStreamConnector implements Connector {
     public GrpcStreamConnector(final FlagdOptions options) {
         channel = ChannelBuilder.nettyChannel(options);
         serviceStub = FlagSyncServiceGrpc.newStub(channel);
+        serviceBlockingStub = FlagSyncServiceGrpc.newBlockingStub(channel);
```

Comment:
Add new stub for calling getSyncMetadata, which uses the same underlying netty channel.